test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1

```diff
@@ -155,49 +155,30 @@ try
         Context "Background jobs within inconsistent mode" {
 
             It "Verifies that background job is denied when mode is inconsistent" {
-
-                try
-                {
+                $e = {
                     $ExecutionContext.SessionState.LanguageMode = "ConstrainedLanguage"
 
                     Start-Job { [object]::Equals("A", "B") }
-                    throw "No Exception!"
-                }
-                catch
-                {
-                    $exception = $_
-                }
-                finally
-                {
-                    Invoke-LanguageModeTestingSupportCmdlet -EnableFullLanguageMode
-                }
+                } | Should -Throw -PassThru
 
-                $exception.FullyQualifiedErrorId | Should -Match "CannotStartJobInconsistentLanguageMode"
+                Invoke-LanguageModeTestingSupportCmdlet -EnableFullLanguageMode
+
+                $e.FullyQualifiedErrorId | Should -Match "CannotStartJobInconsistentLanguageMode"
```

Comment:
Please move to line 162 in `Should -Throw -PassThru `
Below too/

Reply:
Fixed.